pkg/cmd/openshift-controller-manager/controller/build.go

```diff
@@ -52,19 +52,19 @@ func (c *BuildControllerConfig) RunController(ctx ControllerContext) (bool, erro
 		KubeClientExternal:  externalKubeClient,
 		BuildClientInternal: buildClient,
 		DockerBuildStrategy: &buildstrategy.DockerBuildStrategy{
-			Image: c.DockerImage,
+			Image: imageTemplate.ExpandOrDie("docker-builder"),
 			// TODO: this will be set to --storage-version (the internal schema we use)
-			Codec: c.Codec,
+			Codec: annotationCodec,
 		},
 		SourceBuildStrategy: &buildstrategy.SourceBuildStrategy{
-			Image: c.S2IImage,
+			Image: imageTemplate.ExpandOrDie("sti-builder"),
```

Comment:
@bparees is this s2i-builder or sti-builder ?

Reply:
the image name is still "origin-sti-builder" (or ose-sti-builder) so this is correct.